OWNERS

```diff
@@ -1,10 +1,10 @@
 # See the OWNERS docs at https://go.k8s.io/owners
-# See the OWNERS_ALIASES file at https://github.com/kubernetes-sigs/service-apis/blob/master/OWNERS_ALIASES for a list of members for each alias. 
+# See the OWNERS_ALIASES file at https://github.com/kubernetes-sigs/gateway-api/blob/master/OWNERS_ALIASES for a list of members for each alias.
```

Comment:
@bowei Do you have sufficient privileges to do the rename? Just want to make sure we can change the repo name once this PR gets in.

Reply:
This will be completed one https://github.com/kubernetes/org/pull/2491 merges in. The owner aliases we rely on within this repo for approvals are all tied to the local OWNERS_ALIASES file which we simply try to keep in sync with k/org. So even if the timing doesn't match up perfectly, everything should still work.